client/landing/gutenboarding/onboarding-block/question/style.scss

```diff
@@ -69,6 +69,7 @@
 			min-height: 1em;
 			min-width: 60px;
 			transition: color 0.5s ease-in-out 1s, border-color 0.5s ease-in-out 1s;
+			line-height: 1.1; // default is normal (20%)
```

Comment:
This wraps the text in a box of `64px * 1.1`, which is a little closer to the design spec. I am not sure if we can achieve this easily: https://d.pr/i/iYa8lQ It looks a little ambiguous - bottom is 12px from the "line" the characters are sitting on, top is 12px from the topmost letter. 🤔